nuxeo-opencmis-impl/src/main/java/org/nuxeo/ecm/core/opencmis/impl/server/CMISQLtoNXQL.java

```diff
@@ -526,6 +528,13 @@ protected String convertOrderBy(String orderBy, TypeManagerImpl typeManager) {
 
         @Override
         public Boolean walkContains(Tree opNode, Tree qualNode, Tree queryNode) {
+            boolean unfollowCmisSpec = Framework.getService(ConfigurationService.class) //
+                                                .isBooleanPropertyTrue(NuxeoCmisConstants.UNFOLLOW_CMIS_SPEC);
+
+            if (unfollowCmisSpec) {
+                log.warn("You're not following the spec CMIS 1.1. This will potentially cause query parsing error.");
```

Comment:
Agree with you that logging warning if and only if there're more than one `CONTAINS`. By the way, would log level _WARN_ is too high for the user? They're probably aware about the situation when setting `relaxCmisSpec=true`. So should I decrease the log level or simply delete it here?